services/client/client.go

```diff
@@ -0,0 +1,30 @@
+package client
+
+import (
+	"academy/model"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+//ConsultExternalService will call the Jokes API
+func ConsultExternalService() []model.Joke {
+	url := "https://official-joke-api.appspot.com/random_ten"
+	request, error := http.NewRequest("GET", url, nil)
+	if error != nil {
+		fmt.Println(error)
```

Comment:
Even if error is not strictly a reserved word on Go, we always name our errors like `err`, because `error` can and is used as a data type

Also don't forget to return if you catch some error, you would also like to return the error to let the previous layer know that something went wrong, Go can return any number of values from functions

Reply:
Modified to avoid using error